cache/clock_cache.h

```diff
@@ -274,14 +278,18 @@ struct ClockHandle {
 
   std::atomic<uint32_t> refs;
 
+  // True iff the handle is allocated separately from hash table.
+  bool dangling;
```

Comment:
I don't like "dangling" to describe this kind of handle, because something "dangling" is loosely attached. To me that sounds more like erased entries. How about "stopgap"?

Reply:
Fine with me. To me, "stopgap" is more associated with rare and degraded functionality (can't lookup the entry) than "detached." But not a big difference.